### ISSUE TITLE
Headers::send() added ":" to modified Status header.

### ISSUE THREAD

**Issue by dimak08** [phalcon/cphalcon#12179]
$response->setStatusCode(304) add ":" to end of status header

```php
$response = new \Phalcon\Http\Response();
$response->setStatusCode(304);
$response->sendHeaders();
```

![2016-08-30 12 46 25](https://cloud.githubusercontent.com/assets/5196470/18084403/cd266f0a-6eaf-11e6-993b-ac8e4e74601d.png)

### Details

* Phalcon version: 3.0.0 
* PHP Version: 7.0.10

**Comment by Jurigag** [comment]
Yea, i have this too.

**Comment by stamster** [comment]
This is only case with `304 HTTP status`? I'm heavily using `setStatusCode()` method, and never had such issue with 4xx, 5xx or 3xx status codes.

**Comment by dimak08** [comment]
With all statuses except 200

30 авг. 2016 г. 16:13 пользователь "stamster" <notifications@github.com>
написал:

**Comment by stamster** [comment]
You're correct. 

```
HTTP/1.1 404 Not Found:
Server: nginx
Date: Tue, 30 Aug 2016 13:20:05 GMT
Content-Type: application/json; charset=UTF-8
Content-Length: 160
Connection: keep-alive
```

This is stupid bug. xD

**Comment by stamster** [comment]
Response object is set correctly, `var_dump($this->response->getStatusCode()); `

`string(24) "422 Unprocessable Entity"`

**Comment by dimak08** [comment]
I think what bug here: https://github.com/phalcon/cphalcon/blob/master/phalcon/http/response/headers.zep#L90

**Comment by andresgutierrez** [comment]
Fixed in 3.0.x branch

**Comment by stamster** [comment]
@dimak08 I just wanted to point out the same code line number :) You were faster :+1: 

But, really if you look at code this does not look as a bug as someone obviously put effort to make sure `:` will be present on each http status message?! I wonder why.

**Comment by stamster** [comment]
When we can expect 3.0.2 to be released in order to fix this issue? 
@andresgutierrez @sergeyklay

**Comment by sergeyklay** [comment]
@stamster https://github.com/phalcon/cphalcon/milestone/22

**Comment by stamster** [comment]
It would be nice if we could have quick fix for this issue, since it may block some ancient environments (Java 1.6 xD) which refuse to work if they don't find <CR><LF> after HTTP status response.

**Comment by sergeyklay** [comment]
@stamster 
Could you please check 3.0.x branch?

```sh
git clone git@github.com:phalcon/cphalcon.git
cd cphalcon
git checkout 3.0.x

zephir fullclean
zephir build
```

**Comment by Jurigag** [comment]
It's already fixed in 3.0.x
